package.json

```diff
@@ -57,37 +71,28 @@
     "@types/jest": "25.2.1",
     "@types/lodash": "4.14.150",
     "@types/nodemailer": "6.4.0",
-    "@types/pug": "2.0.4",
     "@typescript-eslint/eslint-plugin": "2.29.0",
     "@typescript-eslint/parser": "2.29.0",
-    "handlebars": "4.7.6",
     "husky": "4.2.5",
-    "lint-staged": "10.1.7",
     "jest": "25.4.0",
+    "lint-staged": "10.1.7",
     "nodemailer": "6.4.6",
     "prettier": "2.0.5",
-    "pug": "2.0.4",
-    "ejs": "3.1.2",
     "reflect-metadata": "0.1.13",
     "rimraf": "3.0.2",
     "standard-version": "7.1.0",
     "ts-jest": "25.4.0",
     "ts-node": "8.9.0",
     "typescript": "3.8.3"
   },
-  "lint-staged": {
-    "*.ts": [
-      "prettier --write"
-    ]
-  },
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint -c .commitlintrc.json -E HUSKY_GIT_PARAMS",
-      "pre-commit": "lint-staged"
-    }
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nest-modules/mailer.git"
+  "peerDependencies": {
```

Comment:
I would suggest that only the dependency changes be updated in this diff - it appears there are a few unrelated diffs as well.